dialogflow-add.js

```diff
@@ -1,32 +1,21 @@
-//This is the upload file of google.
-//It first read the data file and transfer it to the upload file.
-//Because of lacking the permission to certain module of nodejs,
-//invoke the python file to finish it.
-
-
 'use strict';
 function createIntents() {
-    const grpc = require('grpc');
-    const prompt = require('prompt');
     this.uploadPhase =function(Intent,projectId){
```

Comment:
For parameters, usually we don't name it starting with capitalized characters. Recommend to change it to `intent`